pkg/apis/v1alpha1/subnet_types.go

```diff
@@ -54,6 +60,44 @@ type SubnetList struct {
 	Items           []Subnet `json:"items"`
 }
 
+// AdvancedConfig is Subnet advanced configuration.
+type AdvancedConfig struct {
+	// StaticIPAllocation configuration for subnet ports with VIF attachment.
+	StaticIPAllocation StaticIPAllocation `json:"staticIPAllocation,omitempty"`
+}
+
+// StaticIPAllocation is static IP allocation for Subnet ports with VIF attachment.
+type StaticIPAllocation struct {
+	// Enable or Disable static ip allocation for Subnet ports with VIF attachment.
```

Comment:
`Enable or disable static IP allocation for subnet ports with VIF attachment.`

Reply:
Done, thanks.